src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -453,7 +453,10 @@ private long GetMaxTombstonesEtagToDelete(TransactionOperationContext context, s
                     || lastRaftIndexBlittable.TryGet(nameof(PeriodicBackupStatus.LastEtag), out long? lastRaftIndex) == false)
                     continue;
 
-                if (lastIncrementalBackupDate == null || lastRaftIndex == null)
+                if (localBackup.TryGet(nameof(PeriodicBackupStatus.LastFullBackup), out DateTime? lastfullBackupDate) == false)
```

Comment:
I think we would want to have an identical code to 5.0 here.
cherry-picking will be problematic here, so simply copy-paste?

note that in 5.0 we don't check for `LastIncrementalBackup` at all.
@ppekrol thoughts?

Reply:
Is this code problematic? What case is not covered?

Reply:
If we have only full backup